src/main/kotlin/com/kotlin/boardproject/domain/comment/service/CommentServiceImpl.kt

```diff
@@ -1,30 +1,30 @@
-package com.kotlin.boardproject.service
-
-import com.kotlin.boardproject.common.enums.ErrorCode
-import com.kotlin.boardproject.common.enums.NotificationType
-import com.kotlin.boardproject.common.enums.PostStatus
-import com.kotlin.boardproject.common.exception.ConditionConflictException
-import com.kotlin.boardproject.common.util.log
-import com.kotlin.boardproject.dto.comment.BlackCommentRequestDto
-import com.kotlin.boardproject.dto.comment.BlackCommentResponseDto
-import com.kotlin.boardproject.dto.comment.CancelLikeCommentResponseDto
-import com.kotlin.boardproject.dto.comment.CreateCommentRequestDto
-import com.kotlin.boardproject.dto.comment.CreateCommentResponseDto
-import com.kotlin.boardproject.dto.comment.DeleteCommentRequestDto
-import com.kotlin.boardproject.dto.comment.DeleteCommentResponseDto
-import com.kotlin.boardproject.dto.comment.LikeCommentResponseDto
-import com.kotlin.boardproject.dto.comment.MyCommentResponseDto
-import com.kotlin.boardproject.dto.comment.UpdateCommentRequestDto
-import com.kotlin.boardproject.dto.comment.UpdateCommentResponseDto
-import com.kotlin.boardproject.dto.notification.NotificationCreateDto
-import com.kotlin.boardproject.model.BlackComment
-import com.kotlin.boardproject.model.Comment
-import com.kotlin.boardproject.model.LikeComment
-import com.kotlin.boardproject.repository.BasePostRepository
-import com.kotlin.boardproject.repository.BlackCommentRepository
-import com.kotlin.boardproject.repository.CommentRepository
-import com.kotlin.boardproject.repository.LikeCommentRepository
-import com.kotlin.boardproject.repository.UserRepository
+package com.kotlin.boardproject.domain.comment.service
+
+import com.kotlin.boardproject.global.enums.ErrorCode
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end